### PR TITLE
[Imagery] Update to be compatible with VIPER

### PR DESCRIPTION
**Pull request by davetsay**
## Overview
Addresses #3748 

## Testing Instructions 
see #3748 

## Author Checklist
| | |
| --- | :---: |
| Changes address original issue? | Y |
| Unit tests included and/or updated with changes? | N/A |
| Command line build passes? | Y |
| Changes have been smoke-tested? | Y |
| Testing instructions included? | Y |

### CODE REVIEW

File: src/plugins/imagery/components/Compass/utils.js [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-
-export function normalizeCompassDirection(degrees) {
-    const base = degrees % 360;
-
-    return base >= 0 ? base : 360 + base;
+    return normalizeCompassDirection(rotation);
 }
 
 export function inRange(degrees, [min, max]) {
+    const point = rotate(degrees);
```
Comment (davetsay) [inline src/plugins/imagery/components/Compass/utils.js:17]: This fixes raw data sun headings like -140 when our compass range is 0 to 360.

File: src/plugins/imagery/components/Compass/utils.js [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
     return min > max
-        ? (degrees >= min && degrees < 360) || (degrees <= max && degrees >= 0)
-        : degrees >= min && degrees <= max;
+        ? (point >= min && point < 360) || (point <= max && point >= 0)
+        : point >= min && point <= max;
 }
 
 export function percentOfRange(degrees, [min, max]) {
-    let distance = degrees;
+    let distance = rotate(degrees);
```
Comment (davetsay) [inline src/plugins/imagery/components/Compass/utils.js:25]: This fixes raw data sun headings like -140 when our compass range is 0 to 360.

File: src/plugins/imagery/components/Compass/utils.js [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
+ * @param  {...number} rotations in degrees
+ * @returns {number} normalized sum of all rotations - [0, 360) degrees
+ */
+export function rotate(...rotations) {
     const rotation = rotations.reduce((a, b) => a + b, 0);
 
-    return normalizeCompassDirection(direction + rotation);
-}
-
-export function normalizeCompassDirection(degrees) {
```
Comment (davetsay) [inline src/plugins/imagery/components/Compass/utils.js:7]: Just moved to bottom since we're not exporting this anymore.

**Review by akhenry — APPROVED** [review]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Pull request merged**